src/api/WorkspaceConfig.ts

```diff
@@ -1,11 +1,20 @@
+import { ServiceDefinition } from '@scalecube/scalecube-microservice/lib/api';
+
 interface WorkspaceConfig {
   name: string;
-  services: ServiceConfig[]
+  services: ServiceConfig[];
+  components: ComponentConfig[];
```

Comment:
```
  components: {
    layout: Layout,
    items: {
      [id: string]: Item
    }
  }
```
`Layout` should contain name, nodeSelector, path and config:any
`Item` should contain name, path, config:any.